src/main/java/com/google/devtools/build/lib/rules/cpp/CppOptions.java

```diff
@@ -994,6 +994,22 @@ public Label getMemProfProfileLabel() {
               + " there are no guarantees about incompatible changes to it or even keeping the"
               + " support in the future. Consider those risks when using it.")
   public boolean experimentalCppModules;
+  @Option(
+      name = "experimental_cpp_modules_with_two_phase_compilation",
```

Comment:
Could this be a feature instead of an option? That would make it more flexible and also require less non-Starlark code.